src/types.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
     badge: figures.ellipsis,
     color: 'blue',
     label: 'waiting',
-    logLevel: 'info'
+    logLevel: 'debug'
```

Comment:
Are you sure about these changes ? 
Looks like info, fav, sucess and other types should use info level by default

Reply:
I'm a bit confused here. Obviously you're right this should be info. It makes no sense that the warn is in debug, and I don't understand what the hell this changes are. 

https://github.com/klaussinani/signale/blame/395c3457e3b166bb3e484994eeae7efcbfcefd69/src/types.js

Here I can see the changes that I recall committing: info, and success are in info level (fav is in debug though, but I recall making this change). So am I too tired to see what is going on here, or is it the github pr view which is broken ?